packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -118,6 +118,21 @@ contract SortedOracles is ISortedOracles, Ownable, Initializable {
     }
   }
 
+  /**
+   * @notice Check if last report is expired.
+   * @param token The address of the token for which the Celo Gold exchange rate is being reported.
+   */
+  function isReportActive(address token) external view returns (bool) {
```

Comment:
The purpose of this function would be more clear with a name that includes the word "expired", like: `isOldestReportExpired`, `expiredReportsExist`, `oldestReportExpired`, etc

Unless we're planning to introduce other qualities that could make a report "active" or "inactive", I think we should stick with "expired" as the adjective here.

Reply:
+1 to `isOldestReportExpired`